SpatialGDK/Source/SpatialGDKFunctionalTests/Public/LogSpatialFunctionalTest.h

```diff
@@ -5,4 +5,4 @@
 #include "Logging/LogMacros.h"
 
 // Intended for logging from test implementations
-DECLARE_LOG_CATEGORY_EXTERN(LogSpatialFunctionalTest, Log, All);
+SPATIALGDKFUNCTIONALTESTS_API DECLARE_LOG_CATEGORY_EXTERN(LogSpatialFunctionalTest, Log, All);
```

Comment:
Potentially debatable whether other modules should be allowed to reference and thus log to this category.

Reply:
This does seem a little questionable. Perhaps we should just add log categories in the project itself. Looks like this category is only used twice in the project.

Reply:
It's a new thing I added along with the tests, since I was running into situations where I wanted to log out more info from places that can't assert via the test framework. So I'd  argue to keep it as part of the test framework itself, but having a new category for the test gyms side would be sensible.

Reply:
Reverted this and added a project-level log category for tests.